### DISCUSSION TITLE
示例文章 article/example-1

### DISCUSSION THREAD

**Discussion by giscus** [tangly1024/tangly1024#21] [Announcements]
# article/example-1

这是示例的文章摘要，摘要内容可被用做搜索，预览。文章列表默认展示概要；若要在列表直接展示文章内容，可在blog.config.js中配置。

https://preview.tangly1024.com/article/example-1

**Comment by tangly1024** [comment]
大家好~

**Reply by tangly1024** [reply]
这是回复

**Comment by txs** [comment]
滾動目錄很讚！

**Comment by Pylogmon** [comment]
系统开启黑暗模式的时候，无论网页是深色模式还是浅色模式，giscus评论区总是有这个底色
![图片](https://user-images.githubusercontent.com/59004461/218304729-c82df826-20d7-4482-8af9-0c890af3fb22.png)
